stories/Organism/Sidebar/sidebar.scss

```diff
@@ -63,7 +63,8 @@
     &__menu {
       border-bottom: none;
       display: block;
-      font-family: $Font-Family-ProximaBold;
+      font-family: $Font-Family-ProximaNova;
```

Comment:
@harishkumar-srijan Do we need the font family here

Reply:
Done